plugin/trino-hive/src/main/java/io/trino/plugin/hive/parquet/ParquetPageSourceFactory.java

```diff
@@ -237,8 +237,8 @@ public static ReaderPageSource createPageSource(
             List<TupleDomain<ColumnDescriptor>> parquetTupleDomains;
             List<TupleDomainParquetPredicate> parquetPredicates;
             if (options.isIgnoreStatistics()) {
-                parquetTupleDomains = ImmutableList.of();
-                parquetPredicates = ImmutableList.of();
+                parquetTupleDomains = ImmutableList.of(TupleDomain.all());
+                parquetPredicates = ImmutableList.of(buildPredicate(requestedSchema, TupleDomain.all(), descriptorsByPath, timeZone));
```

Comment:
```suggestion
                parquetTupleDomains = ImmutableList.of();
                parquetPredicates = ImmutableList.of();
```

Reply:
what am i missing, @findinpath ?

Reply:
If the `parquetTupleDomains` is empty, we're not adding anything in `blocks` and `blockStarts` and `columnIndexes`.
So we're not reading anything from Parquet.